src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,480 @@
+* Aggregate using `average()`, `max()`, `min()`, `sum()`, and `count()`
+* Use `SINCE` and `UNTIL`
+* Plot line graphs with `TIMESERIES`
+* Filter using `WHERE`
+* Group attributes using `FACET`
+
+Want to learn NRQL within the New Relic platform? We've included a NRQL Lessons application on our [opensource.newrelic.com](https://opensource.newrelic.com/projects/newrelic/nr1-learn-nrql) site to help you quickly see NRQL's value and power within the application. The video below shows you how to install the NRQL Lesson.
+
+<Video
+  id="3UHEHbprs6I"
+  type="youtube"
+/>
+
+Ready to get started? Let's go!
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="first-query"
+    title="1. Your First Query"
+  >
+  <SideBySide>
+  <Side>
+Let's start by looking at a single event type in NRDB called **Transaction**, gathered by New Relic APM. Every NRQL query must have `SELECT` and `FROM` clauses: you must `SELECT` some data and tell us where it's `FROM`. Start with this basic query that selects everything from the **Transaction** event type.
+ 
+    ```sql
+    SELECT *
+    FROM Transaction
+    ```
+
+  </Side>
+  <Side>
+    <img
+  title="Select and From query"
+  alt="A screenshot of a query with Select and From"
+  src={queriesnrqltutorial1}
+/>
+  </Side>
+  </SideBySide>
+  <SideBySide>
+  <Side>
+ This returns a lot of results, each with a timestamp and a collection of attributes. For now, we only want a single result, so let's limit the results to a single record using `LIMIT 1`.
+
+ <Callout variant="important"> When a LIMIT isn't supplied the default will be used, which is 100 table rows for SELECT * queries or 10 aggregated values for FACET queries and SELECT (attributes) queries. You can specify any limit up to the maximum. Use LIMIT MAX to return the maximum number of results possible.
+</Callout>
+
+    ```sql
+    SELECT *
+    FROM Transaction
+    LIMIT 1
+    ```
+  </Side>
+  <Side>
+ <img
+  title="Select, From, and Limit query"
+  alt="A screenshot of a query with Select, From, and Limit"
+  src={queriesnrqltutorial2}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    We're now controlling the volume of the results. But what if we don't want all the attributes? What if we would prefer to see only specific data points? Like SQL, we can do this with only a few characters by replacing * with the name of the attribute(s) we want. In this case, we'll ask for the name of a transaction and the duration of time it took.
+
+    ```sql
+    SELECT name, duration
+    FROM Transaction
+    ```
+    
+  </Side>
+  <Side>
+  <img
+  title="Select and From with Duration query"
+  alt="A screenshot of a query with Select and From using Duration"
+  src={queriesnrqltutorial3}
+/>
+  </Side>
+</SideBySide>
+With this, you've completed queries that can return either all or some specific attributes of an event, as well as querying a specific number of events. Let's go to the next step: aggregating data.
+  </Collapser>
```

Comment:
I wanted to have a better separation too! I was thinking of asking Eng to see if something can be done about that as well as a few other things after everyone is back from the holidays.

Reply:
@jeff-colucci Please do ask the eng team if they can help with this. I also really like this side-by-side approach. I don't know if this is a better option, but I kind of wanted to see the screenshots next to the codeblocks rather than the explanation.

OR I'd want to see the screenshot on the left and the explanation on the right. This might help break things up a little bit given the current limitations?

Reply:
I like the idea of swapping the images and text. I thought that the query examples before the explanations were a little wonky (which would fix the positioning issue), but maybe there is a way or there can be a way to center or bottom justify the images in the side-by-sides which would match the positioning.